scss/_navbar-offcanvas.scss

```diff
@@ -333,6 +339,7 @@
       position: absolute;
       .dropdown-item {
         padding: $dropdown-item-padding-y $dropdown-item-padding-x;
+        white-space: nowrap;
```

Comment:
This causes the menu to get cut off with long menu titles. Looking into a solution now (we also had to fix this in QS1) but wanted to ensure it was shared before this got merged.

Quickest solution would be to not include this until a solution for all screen sizes is discovered. Not including it just makes the dropdown menu skinnier than desired for now.

![image](https://user-images.githubusercontent.com/59745109/110531900-65da5c80-80d9-11eb-975f-fd410353aebe.png)